test_cli_backend_basic.py

```
@@ -0,0 +1,145 @@
+async def test_basic_execution():
+    """Test basic CLI backend execution (mock)."""
+    print("Testing basic CLI backend execution...")
+    try:
+        from praisonai.cli_backends import resolve_cli_backend
+        from praisonaiagents import CliSessionBinding
+        
+        backend = resolve_cli_backend("claude-code")
+        
+        # Test basic structure
+        result = await backend.execute("Hello, test!", session=CliSessionBinding(session_id="test"))
+        print(f"✓ CLI backend execution completed")
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)